Extensions/xkit_patches.js

```diff
@@ -2435,61 +2435,10 @@ XKit.extensions.xkit_patches = new Object({
 			}
 		};
 
-		// Expedited fix for #1509; revert after updated XKit extension has been widely installed.
```

Comment:
this should not be removed for security issues, seeing as more than a few users refuse to upgrade to firefox quantum. happy to merge the additions as they are if this removal is.. removed

Reply:
Riiiiiiiiight, sgtm

Reply:
(updooted accordingly)